rules/gcs.bzl

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Are you using this rule anywhere in this PR? if not, split it off.

Reply:
Will create a separate PR for this file, remove it from current PR.